src/core/ext/transport/cronet/transport/cronet_transport.c

```diff
@@ -51,612 +51,918 @@
 
 #define GRPC_HEADER_SIZE_IN_BYTES 5
 
-// Global flag that gets set with GRPC_TRACE env variable
+#define CRONET_LOG(...)                          \
+  {                                              \
+    if (grpc_cronet_trace) gpr_log(__VA_ARGS__); \
+  }
+
+/* TODO (makdharma): Hook up into the wider tracing mechanism */
 int grpc_cronet_trace = 1;
 
-// Cronet transport object
+enum OP_RESULT {
+  ACTION_TAKEN_WITH_CALLBACK,
+  ACTION_TAKEN_NO_CALLBACK,
+  NO_ACTION_POSSIBLE
+};
```

Comment:
Is it ok for `ACTION_TAKEN_WITH_CALLBACK` to be the default? It'll be assigned the value of 0, and we usually memset data structures to zero. Perhaps an uninitialized value would make sense, but I don't know enough about where this enum is used.